pages/_document.tsx

```diff
@@ -113,6 +115,7 @@ class MyDocument extends Document {
           <link rel="manifest" href="/manifest.json" />
         </Head>
         <body>
+          <ColorModeScript initialColorMode={theme.config.initialColorMode} />
```

Comment:
```suggestion
          <ColorModeScript initialColorMode={"system"} />
```

`pages/_app.tsx ` でもColorModeScriptを読んでるんですが、こっちを変えるべきかもしれないです